src/genie/libs/parser/iosxe/show_hw_module_status.py

```diff
@@ -0,0 +1,116 @@
+''' show_hw_module_status.py
+IOSXE parsers for the following show commands:
+    * show hw_module subslot {subslot} transceiver {transceiver} status
+'''
+
+import re
+
+from genie.metaparser import MetaParser
+# from genie.metaparser.util.schemaengine import Any, Optional
+
+# parser utils
+# from genie.libs.parser.utils.common import Common
+
+
+# ==========================
+# Schema for:
+#  * 'show_hw_module_status'
```

Comment:
can you put the actual show command here

Reply:
Added the actual command.